_internals/system-views.md

```diff
@@ -4,47 +4,47 @@ title: Babelfish system view
 nav_order: 4
 ---
 
-## Babelfish system views 
+## Babelfish system views
 
-Once in a while, an application has to access an MS SQL-compatible system view to
+Once in a while, an application has to access an MS SQL-compatible system view
+or a server side function to
 work properly. This is especially true for graphical user interfaces. However,
-accessing an MS SQL-compatible system view is also necessary for some applications 
+accessing an MS SQL-compatible system view is also necessary for some applications
 which have to collect information from the database to function properly.
 
 Providing support for system views is therefore important. The following views
 are currently available:
 
-- `sys.sysdatabases`
-- `sys.pg_namespace_ext `
-- `sys.server_principals`
-- `sys.databases`
-- `sys.tables `
-- `sys.views `
 - `sys.all_columns`
-- `sys.all_views `
-- `sys.columns `
+- `sys.all_objects`
+- `sys.all_views`
+- `sys.columns`
+- `sys.databases`
 - `sys.foreign_key_columns`
-- `sys.foreign_keys `
-- `sys.identity_columns `
-- `sys.indexes `
-- `sys.key_constraints `
+- `sys.foreign_keys`
+- `sys.identity_columns`
+- `sys.indexes`
+- `sys.key_constraints`
+- `sys.objects`
+- `sys.pg_namespace_ext`
 - `sys.procedures`
-- `sys.schemas `
+- `sys.schemas`
+- `sys.server_principals`
+- `sys.sp_columns_100_view`
+- `sys.sp_tables_view`
+- `sys.spt_columns_view_managed`
+- `sys.spt_tablecollations_view`
 - `sys.sql_modules`
-- `sys.sysforeignkeys `
-- `sys.sysindexes `
-- `sys.sysprocesses `
-- `sys.types `
-- `sys.objects `
-- `sys.sysobjects `
-- `sys.all_objects `
-- `sys.system_objects `
-- `sys.sysmail_allitems`
-- `sys.sysmail_faileditems`
-- `sys.sysmail_mailattachments`
-- `sys.sysmail_sentitems`
-- `sys.sysmail_unsentitems`
 - `sys.syscharsets`
+- `sys.sysdatabases`
+- `sys.sysforeignkeys`
+- `sys.sysindexes`
+- `sys.sysobjects`
+- `sys.sysprocesses`
+- `sys.system_objects`
+- `sys.tables`
+- `sys.types`
+- `sys.views`
 
```

Comment:
This list is wrong. For one, catalogs whose names start with 'sys' should not be prefixed with 'sys.' (there's a bigger story, but 'sys.' is best left off).  Also, try selecting from spt_columns_view_managed and you'll see why it should not be in this list. 
In short, only the following SQL Server catalogs are supported, the rest is no (whether or not these object may contain in the source code does not matter - its whether they work from a customer perspective):

sys.all_columns
sys.all_objects
sys.all_views
sys.assemblies
sys.assembly_types
sys.columns
sys.databases
sys.extended_properties
sys.foreign_key_columns
sys.foreign_keys
sys.indexes
sys.key_constraints
sys.objects
sys.procedures
sys.schemas
sys.server_principals
sysobjects
sys.tables
sys.view
syscharsets
sysdatabases
sysforeignkeys

Reply:
Okaaay. If you say so.

I felt like contradicting, on the same grounds that we should document all GUCs, but since the chapert now starts with "You can query the following system catalog views", you have a point.

I'll add a sentence to hint at the existence of other catalog tables.